components/map/AGHLeaveIndicator.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+function AGHLeaveIndicator() {
+  return (
+    <div className="pointer-events-none absolute left-0 top-0 box-border size-full border-4 border-red-600">
```

Comment:
Zmienić z border-red-600 na border-errorAlert lub border-errorAlert-foreground 

Ale to drugie chyba lepsze